docs/package.json

```diff
@@ -5,17 +5,17 @@
   "license": "Apache-2.0",
   "homepage": "https://pinterest.github.io/gestalt",
   "scripts": {
-    "build": "webpack --config webpack.config.corkboard.js --colors --progress --output-path build",
-    "start": "webpack-dev-server --config webpack.config.corkboard.js --progress --colors --open"
+    "build": "webpack --config webpack.config.js --colors --progress --output-path build",
+    "start": "webpack-dev-server --config webpack.config.js --progress --colors --open"
   },
   "dependencies": {
-    "corkboard": "0.3.3",
     "gestalt": ">0.0.0",
     "highlight.js": "^9.12.0",
     "marked": "^0.3.17",
-    "react": "^15.4.1",
     "react-dom": "^15.4.1",
-    "react-live": "^1.10.1"
+    "react-live": "^1.10.1",
+    "react-router": "^2.0.1",
```

Comment:
doesn't really matter since this is docs, but just curious why 2?

Reply:
Because it was the same version as corkboard had. Can do an upgrade in another PR.